.github/workflows/deploy-cpp.yml

```diff
@@ -64,10 +69,21 @@ jobs:
       with:
         submodules: true
 
+    - name: Python 3.10
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+
     - name: Compute SOURCE_DATE_EPOCH
+      shell: python
```

Comment:
That's new to me. I guess you opted for Python as a shell because to use `strip()`. There's probably a shell equivalent, but why bother? `:)`